src/Momento.Sdk.Incubating/Requests/CollectionTtl.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+namespace Momento.Sdk.Incubating.Requests
+{
+    /// <summary>
+    /// Represents the desired behavior for managing the TTL on collection
+    /// objects (dictionaries, lists, sets) in your cache.
+    ///
+    /// For cache operations that modify a collection, there are a few things
+    /// to consider.  The first time the collection is created, we need to
+    /// set a TTL on it.  For subsequent operations that modify the collection
+    /// you may choose to update the TTL in order to prolong the life of the
+    /// cached collection object, or you may choose to leave the TTL unmodified
+    /// in order to ensure that the collection expires at the original TTL.
+    ///
+    /// The default behavior is to refresh the TTL (to prolong the life of the
+    /// collection) each time it is written.  This behavior can be modified
```

Comment:
It might be a good idea to mention that refreshing TTL is our default behavior in `README`?

Reply:
That's an interesting idea.  This only applies to collection operations though, and only to "write" operations at that.  We don't currently have any specific treatment of collection operations in the README so I'm not sure where I would fit it in without it being a bit awkward.

Also I'm very much hoping that users will just assume that this is the default behavior because it seems like the most obvious / logical behavior to me.  So I think maybe let's keep an eye out to see if we notice any users stumbling on it, and if we do, then we should definitely add a section to the README.  wdyt?

Reply:
When I think about it now, I'm not sure when you don't want to refresh ttl if you're adding new items...
When you don't want old items to stay in the same cache with the new items...?
But then you would delete those old items and then add the new items with a refreshed ttl..?

Anyways, I'm okay with your proposal!